templates/controller-deployment.yaml

```diff
@@ -59,10 +59,14 @@ spec:
         - "-ec"
         - |
           consul-k8s controller \
+            {{- if .Values.controller.logLevel }}
+            -log-level={{ .Values.controller.logLevel }} \
+            {{- else }}
+            -log-level={{ .Values.global.logLevel }} \
+            {{- end }}
             -webhook-tls-cert-dir=/tmp/controller-webhook/certs \
             -datacenter={{ .Values.global.datacenter }} \
             -enable-leader-election \
```

Comment:
`-log-json` isn't passed

Reply:
Ah good catch, actually this was intentional because it wasn't initially scoped in the PR, but I've since added that so it definitely needs to go in to both controller and inject-connect!